frontend/src/components/Navbar/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './index.scss';
+import logo from '../../assets/images/Navbar images/Group 289.png';
```

Comment:
przy konfiguracji projektu ustawiłem folder bazowy jak src, wiec w przypadku importów wystarczy coś takiego jak poniżej
```suggestion
import logo from 'assets/images/Navbar images/Group 289.png';
```